l1/internal/contract/starknet.go

```diff
@@ -0,0 +1,103 @@
+// Code generated - DO NOT EDIT.
```

Comment:
how was it generated?

Reply:
Geth's [abigen](https://geth.ethereum.org/docs/tools/abigen) utility. 
> Abigen creates easy-to-use, type-safe Go packages from Ethereum smart contract definitions known as ABIs. This abstracts away a lot of the complexity of handling smart contract deployment and interaction in Go native applications such as encoding and decoding smart contracts into EVM bytecode

```
abigen --abi coreContract_0.11.1.json --pkg starknet > starknet.go
```